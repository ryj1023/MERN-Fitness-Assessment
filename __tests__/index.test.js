import React from 'react'
import Home from '../pages'
import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'
import { Provider } from 'react-redux'
import { createStore } from 'redux'

describe('Home Page component', () => {
    const initialState = { output: 100 }
    const mockStore = configureStore()
    let store, wrapper
    beforeEach(() => {
        store = mockStore(initialState)
        wrapper = shallow(<Home store={store} />)
    })
    it('expect to render Home component', () => {
        expect(wrapper).toMatchSnapshot()
    })
})
