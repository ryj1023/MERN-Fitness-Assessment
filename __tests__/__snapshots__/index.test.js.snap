// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page component expect to render Home component 1`] = `
<ContextProvider
  value={null}
>
  <Home
    getDailyDietGoals={[Function]}
    getFeaturedRecipeList={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  />
</ContextProvider>
`;
